WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/AddressSelectorCommand.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+import Yosemite
+import SwiftUI
+
+protocol AddressSelectorCommandProtocol {
+    var name: String { get }
+    var code: String { get }
+}
+
+extension Country: AddressSelectorCommandProtocol {}
+extension StateOfACountry: AddressSelectorCommandProtocol {}
+
+final class AddressSelectorCommand: ObservableListSelectorCommand {
+    /// Original array of countries or states.
+    ///
+    private let countriesOrStates: [AddressSelectorCommandProtocol]
```

Comment:
Since we might want to extend this, I would go for a more generic name than `countriesOrStates` for this property. What about `areas`?

Reply:
Sounds good! Changed on https://github.com/woocommerce/woocommerce-ios/pull/7713/commits/4caaa7c8fb6086c837c992223170554cd560a2c1